lib/embetty.js

```diff
@@ -34,7 +35,8 @@ module.exports = class Embetty {
     const response = await this.get(url, options)
     return {
       data: response.body,
-      type: contentType.parse(response).type,
+      type: 'image/jpeg'
```

Comment:
I had to replace the type from **contentType.parse(response).type** to **image/jpeg** because of error **TypeError: content-type header is missing from object**.